aten/src/ATen/core/jit_type.h

```diff
@@ -804,17 +804,18 @@ struct TORCH_API TensorType : public SharedType {
       return strides;
     };
 
-    std::vector<int64_t> dim_order(sizes.size());
-    if (memory_format == MemoryFormat::ChannelsLast) {
+    auto ndims = sizes.size();
+    std::vector<int64_t> dim_order(ndims);
+    if (memory_format == MemoryFormat::ChannelsLast && ndims == 4) {
```

Comment:
Im curious, why is this change needed? I mean, it does look correct, but if something failed without it, it would be nice to add it as a regression test.